oda_api/api.py

```diff
@@ -930,10 +930,11 @@ def __init__(self, data_list, add_meta_to_name=['src_name', 'product'], instrume
 
             name, var_name = self._build_prod_name(
                 data, name, add_meta_to_name)
-            setattr(self, var_name, data)
+            # TODO what is the reason for this ?
+            # setattr(self, var_name, data)
```

Comment:
This makes the data available as attributes of the object.
One may argue how convenient it is, surely it is sometimes and sometimes it causes issues.

But, as of now, it **must remain** since it is interface used in various notebooks, some of them published.